src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -351,6 +361,14 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
             });
         };
 
+        var setFilterColumns = function(userFilterColumns, inactiveFilterColumns) {
+            // exclude columns which have multiple predicates
+            angular.extend(filterColumns, {
+                userFilterColumns: userFilterColumns.filter(c => (c.predicate === undefined || c.predicate == null) || c.predicate.trim().indexOf(' ') === -1),
```

Comment:
```
c.predicate == null
```
Should probably be `c.predicate === null` like is done with the `undefined` check?

There is also unnecessary quoting.
Adding extra quoting around `||` or `&&` logic helps make code harder to read.
I would think the following is better:
```
c.predicate === undefined || c.predicate == null || c.predicate.trim().indexOf(' ') === -1
```

Reply:
Will update to use strict equality. The quoting was for readability.

Reply:
Looking at this, it is actually not correct as a submission list column predicate can be null per database schema. It should be only excluding submission list column that do have a predicate and it is a combined predicate.

Reply:
Updated.